test/TestProjects/AmbiguousMethods-LowLevel/AmbiguousClient.cs

```diff
@@ -0,0 +1,194 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace AmbiguousMethods_LowLevel
+{
+    /// <summary> Existing Ambiguous service client. Methods are changed to intentionally have ambiguity with DPG. </summary>
+    public partial class AmbiguousClient
+    {
+        private readonly HttpPipeline pipeline = HttpPipelineBuilder.Build(ClientOptions.Default);
+
+        /// <param name="cancellationToken"> CancellationToken to use. </param>
+        public virtual async Task<Response> NoParamAsync(CancellationToken cancellationToken = default)
```

Comment:
Would it be possible to make the return values of convenience methods return Response<T> so it's easy to spot which are the convenience methods and which are the protocol methods in an APIView?

Reply:
I change the return type from `Response` to `Response<SuccessResponse>`.